server/task_executor_schema_update_ghost_sync.go

```diff
@@ -225,12 +229,24 @@ func executeGhost(task *api.Task, statement string, syncDone chan<- struct{}) er
 	defer cancel()
 	migrator := logic.NewMigrator(migrationContext)
 
-	go func(ctx context.Context, migrationContext *base.MigrationContext) {
+	go func(ctx context.Context) {
 		ticker := time.NewTicker(1 * time.Second)
 		defer ticker.Stop()
+		createdTs := time.Now().Unix()
 		for {
 			select {
 			case <-ticker.C:
+				var (
```

Comment:
Let's update the progress to 100% when migration is done.

Reply:
The progress lives as long as the task executor. That the task status is `Done` means we have collected the task executor in the task scheduler and don't have access to the progress. Display the progress as 100% for a completed task can be done on the frontend IMO.

Reply:
SG.